app/assets/src/components/views/report/utils/download.js

```diff
@@ -0,0 +1,73 @@
+import { compact } from "lodash/fp";
+
+const NON_HOST_READS_LABEL = "Download Non-Host Reads (.fasta)";
+const NON_HOST_CONTIGS_LABEL = "Download Non-Host Contigs (.fasta)";
+const NON_HOST_ASSEMBLIES_LABEL = "Download Non-Host Assemblies (.fasta)";
+const UNMAPPED_READS_LABEL = "Download Unmapped Reads (.fasta)";
+const RESULTS_FOLDER_LABEL = "See Results Folder";
+
+// Get download options based on pipeline metadata.
+const getDownloadOptions = (pipelineRun, assembledTaxids) => {
+  let stageTwoComplete = pipelineRun && pipelineRun.adjusted_remaining_reads;
+  let nonhostAssemblyComplete =
+    assembledTaxids && assembledTaxids.indexOf("all") >= 0;
+  const assembled = pipelineRun && pipelineRun.assembled === 1;
+
+  return compact([
+    stageTwoComplete && NON_HOST_READS_LABEL,
+    assembled && NON_HOST_CONTIGS_LABEL,
+    nonhostAssemblyComplete && NON_HOST_ASSEMBLIES_LABEL,
```

Comment:
I'm assuming YF said, but what's the difference between assembly being complete and non-host assembly being complete?

Reply:
I will check with her. This logic is already used in the code base for the buttons on the Sample Details tab, so not going to block this PR on that.